src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyBasedCallerUtils.java

```diff
@@ -801,10 +811,23 @@ else if ( ! phaseSetMapping.containsKey(comp) ){
      * Is this variant bi-allelic?  This implementation is very much specific to this class so shouldn't be pulled out into a generalized place.
      *
      * @param vc the variant context
-     * @return true if this variant context is bi-allelic, ignoring the NON-REF symbolic allele, false otherwise
+     * @return true if this variant context is bi-allelic, ignoring the NON-REF symbolic allele and '*' symbolic allele, false otherwise
      */
     private static boolean isBiallelic(final VariantContext vc) {
-        return vc.isBiallelic() || (vc.getNAlleles() == 3 && vc.getAlternateAlleles().contains(Allele.NON_REF_ALLELE));
+        if (vc.isBiallelic()) return true;
```

Comment:
`return vc.isBiallelic() || vc.getAlternateAlleles().stream().filter(isSiteSpecificAltAllele).count() == 1`

Reply:
Done, but then I realized that the `vc.isBiallelic()` part is redundant with the second part of the test so I removed it.